### ISSUE TITLE
New Aimbot suggestion

### ISSUE THREAD

**Issue by TheGamingTitan11** [ImpactDevelopment/ImpactClient#617]
I have a small suggestion.

I play on this server called GTAMC and it just got revamped and updated with new stuff.the server has guns on it and you have to aim to shoot the gun.In regular, without the texture pack they are just tools likes diamond pickaxes and gold shovels.I wanted to ask for a feature that would automatically aim tools and shoot them for this server.Just like the minigame section dedicated for specific games like minestrike which is only on mineplex, I'm asking for this.Thanks.

Also, i want to donate alot of money to this client and it's development but I can't do it through paypal and i only have PaySafe Gift Cards.is that fine? Or not? Please let me know, thanks.

**Comment by LeafHacker** [comment]
I've updated your issue in the interest of clarity. How exactly do you want the "aim and shoot" functionality to work? Would left-clicking automatically aim? Are you asking for some kinda aimbot?

Regarding donations, that is something you should discus with Brady on [discord](https://discord.gg/YFhR2Ab).

**Comment by TheGamingTitan11** [comment]
Yes, I'm asking for a type of aimbot. I'll list the details of the server.
-With guns, you can left click to zoom in for a better shot, 
-Right clicking with the zoomed or unzoomed gun will shoot it.
-Without the texture pack, the guns look like regular tools.
-Pressing q will reload the weapon
I want it so that it automatically zooms and shoots with snipers and AR's, Shotguns and pistols without having to manually shoot it.

**Comment by TheGamingTitan11** [comment]
Bump*

**Comment by LordQuadrato** [comment]
@TheGamingTitan11 This is not a help site/forum. Please only add relevant information and don't post meaningless comments. Your issue will be reviewed when the developers decide to implement it.

**Comment by TheGamingTitan11** [comment]
When am I acting like this is a help/site forum?I said "Bump" because on alot of forum sites, saying something refreshes it so the developers an respond to it.I said it because nobody responded in a while and I wanted them to get this information.Secondly, please only respond to your business and no one elses.If I get in trouble for something I did wrong, that is my problem, not yours.Thirdly, who are you to tell me to stop?You could have asked me why I posted that comment before immediately deciding it was irrelevant.Respond in a comment giving me feedback, or some critic on my idea.Or if you dont have anything helpful to say, don't say it.Fourthly, don't look at the title of the forum topic that says "issues".It's used for much more than that, including suggestion,bugs,feedback,and help and clarity.

**Comment by LordQuadrato** [comment]
Clearly we have misunderstood each other. I'll explain and clear up a few things.

> When am I acting like this is a help/site forum?

Bumping issues to receive a response is generally a bad practise on issue trackers, because it doesn't contribute anything to the conversation (and also doesn't refresh the issue). It will only send a mail to everyone currently watching this issue or the whole repository.
Brady and Leaf are aware of (almost) all suggestions and bugs. They will look through them when they're developing a new version and then they decide which ones to implement.

>  I said it because nobody responded in a while

Literally only _3 days_ without activity. We also have our daily life and not every issue has the same importance. If you want to discuss your issue so it gets upvoted and reviewed recently, please use [discord](https://discordapp.com/invite/YFhR2Ab "Impacts Discord").

> Thirdly, who are you to tell me to stop? You could have asked me why I posted that comment before immediately deciding it was irrelevant.

My comment might have sounded a little rude, but I only wanted to keep it "plain and simple" and I thought you would understand it.

> Thirdly, who are you to tell me to stop? You could have asked me why I posted that comment before immediately deciding it was irrelevant.

I'm a member of the [Impact Organisation](https://github.com/ImpactDevelopment) and I help to keep the issue tracker clean, answering to questions and reviewing issues. In your case I wanted to prevent you from doing it again and again.

> Or if you don't have anything helpful to say, don't say it

Yes thats exactly what I was saying :+1:

I am going to **hide this conversation** and future comments regarding the "bump" topic, marking conversation as resolved for now. Please don't comment to this pointless topic anymore.
_If you want further discussion, you can ask the community about their opinion on Impacts [discord](https://discordapp.com/invite/YFhR2Ab)_

**Comment by TheGamingTitan11** [comment]
I'll start with this, isn't everyone a part of the Impact community if they have an account registered?
And also, I saw that you were a bit rude, and I was being a bit aggressive so I'll apologize now for my behavior.I would look forward to receiving an apology as well, and then we can continue on without any issues, okay?
